doc-content/enterprise-only/installation/clustering-bc-configure-standalone_proc.adoc

```diff
@@ -13,6 +13,20 @@ These steps describe a basic cluster configuration. For more complex configurati
 * AMQ Broker is installed and configured, as described in <<JMS-broker-configure_{context}>>.
 * {EAP} and {PRODUCT} are installed on each node of the cluster as described in <<clustering-download-eap-bc-proc>>.
 * An NFS server with a mounted partition is available.
++
+[NOTE]
+====
+To configure an NFS server with a mounted partition, perform the following steps:
+
+. Configure the NFS server. For more information, see https://access.redhat.com/solutions/1355233[_How to configure NFS in RHEL 7_].
+. To add the shared folder, enter the following command:
++
+----
+# vi /etc/exports /data/shared * (rw,sync,no_root_squash)
```

Comment:
*(rw,sync,no_root_squash) I think the space needs to be removed

Also, be careful because is a user wants to execute that whole line it will get into troubles because `vi` will thing that all those things are or files or commands, so may I suggest to write `/data/shared *(rw,sync,no_root_squash)` in a new line

Reply:
yes, good points, rechecked the KCS and you're right, making those changes now